lib/broccoli/ember-app-with-packages.js

```diff
@@ -71,7 +71,7 @@ function EmberAppWithPackages(defaults, options) {
         templates: new Funnel(appBuildConfig.appTreesBasePath + '/templates')
       }
     };
-    this.bootApp = new EmberApp(configWithTrees, defaults, sharedBuildConfig, appBuildConfig);
+    this.bootApp = new EmberApp(configWithTrees, sharedBuildConfig, appBuildConfig, defaults);
```

Comment:
Do you know what are the keys/sections in defaults that we need when [creating packages](https://github.com/MiguelMadero/ember-cli-bundle-loader/blob/master/lib/broccoli/ember-app-with-packages.js#L129)? I want to see if there is something else we should be passing explicitly instead. I don't think we should make defaults the last option since that would take precedence and we may intentionally want to overwrite defaults. 

What version of ember-cli are you using? It seems like there was a breaking change somewhere between checked 2.4.2 and 2.7, now the constructor takes defaults and options instead of a set of overrides. I'm thinking that this might be the issue and not so much the order.